stable/traefik/templates/_helpers.tpl

```diff
@@ -123,3 +123,14 @@ Create the block for RootCAs.
 	   {{- end -}}
          ]
 {{- end -}}
+
+Create the block for mTLS ClientCAs.
```

Comment:
I think you are missing an open comment block here

Reply:
Good spot, looks like the block above this one was also missing the same. Have added both